p2p/src/peers/mod.rs

```diff
@@ -109,6 +142,8 @@ impl Peers {
             })
             .collect();
 
+        log::debug!("Added new peers: \n{}", self);
```

Comment:
Shall we change all this log messages to trace?

Reply:
yep, i think so